src/main.js

```diff
@@ -183,52 +197,48 @@ var manifestor = function(options) {
     if (!arguments.length) return _canvasImageStates;
     _canvasImageStates = state;
 
-    // if (!initial) {
-    //     jQuery.publish('annotationsTabStateUpdated' + this.windowId, this.tabState);
-    // }
-
     return _canvasImageStates;
   }
 
-  // function detailTransition(detailLayout) {
-  //     renderLayout(detailLayout);
-  // }
-  // function overviewTransition(selection) {
-  //     renderLayout(detailLayout);
-  // }
   function setScrollElementEvents() {
     var animationTiming = 1200;
     var interactionOverlay = d3.select(overlays[0]);
     if (canvasState().perspective === 'detail') {
       interactionOverlay
         .style('opacity', 0)
-        .transition()
```

Comment:
These transitions don't do anything, since the properties they are "Transitioning" aren't continuous, but discrete.